airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -76,6 +89,7 @@ class Config:
         minimum=1,
         maximum=30,
     )
+    custom_insights: Optional[Json] = Field(description="A json objet with custom insights")
```

Comment:
I think you need to declare a new model here - `InsightConfig` and have a list of it here